Sources/Recoverable/Recoverable.swift

```diff
@@ -31,44 +29,74 @@ extension UIView: Recoverable {
         layer.cornerRadius = safeViewState.cornerRadius
         layer.masksToBounds = safeViewState.clipToBounds
         
+        startTransitionIfAvailable()
+        
         if safeViewState.backgroundColor != backgroundColor || forced {
             backgroundColor = safeViewState.backgroundColor
         }
     }
+    
+    fileprivate func startTransitionIfAvailable() {
+        if let transitionType = currentSkeletonConfig?.transition {
+            switch transitionType {
+            case .none:
+                recover()
+            case .fade(duration: let duration):
+                fadeIn(duration: duration)
+            }
+        }
+    }
+    
+    @objc fileprivate func recover() {}
```

Comment:
why we need this method? each `Recoverable` view has a method to recover the view state, right?

Reply:
Recover is only used if the view should recover without transition.